.github/workflows/ci.yml

```diff
@@ -1,152 +1,84 @@
 name: ci
 on:
+  pull_request:
   push:
     branches:
     - master
-  pull_request:
-    types:
-    - opened
-    - reopened
-    - synchronize
 jobs:
-  linux-tests:
-    name: linux tests
-    runs-on: ubuntu-18.04
+  tests:
+    name: tests
+    strategy:
+      matrix:
+        # os: [ubuntu-18.04, macos-10.14, windows-2016]
+        os: [ubuntu-18.04, macos-10.14]
```

Comment:
```suggestion
        os: [ubuntu-18.04, macos-10.14, windows-2016]
```

Reply:
The tests on windows are failing because of https://github.com/atom/github/pull/2459#issuecomment-624725972

Reply:
Still, we should run them to observe if a new test starts to fail. We should continue to on error to not block the CI. This is much better than not running them.

```yaml
      - name: Run the tests
        if: ${{ !contains(matrix.os, 'windows') }}
        run: atom --test spec

      - name: Run the tests on Windows 
        if: ${{ !contains(matrix.os, 'windows') }}
        continue-on-error: true # due to https://github.com/atom/github/pull/2459#issuecomment-624725972
        run: atom --test spec
```

Reply:
I think a better solution would be to figure out how to fix the 8.3 path names in the tests and run them normally on windows. But as I said above, unless @smashwilson actually wants to merge this PR, it is just a POC.

Reply:
It should be pretty easy to create a function that can compare regular paths with FAT 8.3 paths in tests.

Reply:
Definitely, in the long run, we can fix the test.

Reply:
> unless @smashwilson actually wants to merge this PR, it is just a POC.

I'm :+1: on merging this if one of you has time to get it green!

And I'm 👍🏻 on including the Windows builds until we can fix that crash.